Jenkinsfile

```diff
@@ -602,6 +602,7 @@ podTemplate(name: 'fedora-atomic-inline', label: 'fedora-atomic-inline', cloud:
                         // Set groovy and env vars
                         env.task = "./ci-pipeline/tasks/atomic-host-tests"
                         env.playbook = "ci-pipeline/playbooks/system-setup.yml"
+                        env.ANSIBLE_HOST_KEY_CHECKING = "False"
```

Comment:
I believe this is set for the overall pipeline so it should not to be set again

Reply:
It is set in rpmbuild, ostree-compose, ostree-image-compose, ostree-image-boot-sanity, and ostree-boot-sanity currently. Figured it was worth a try